### PR TITLE
Made code Google Shellcheck complaint

### LINKED ISSUE #27: Ensure shellcheck compliance.
This is the Google Shell style Guide:https://google.github.io/styleguide/shellguide.html die which can be used for Bash. To do so, one can make a list of TODO's per file/function and cross of the list of TODO's for each function:
A. Section: Comments, subsection: File Header: Start each file with a description of its contents.
B. Section: Comments, subsection: Function Comments: Any function that is not both obvious and short must be commented.
C. Section: Comments, subsection: Function Comments: It should be possible for someone else to learn how to use your program by reading the comments without reading the code.
D. Section: Comments, subsection: Function Comments: All function comments should describe the intended API behaviour using:
D.0 Description of the function.
D.1 Globals: List of global variables used and modified.
D.2 Arguments: Arguments taken.
D.3 Outputs: Output to STDOUT or STDERR.
D.4 Returns: Returned values other than the default exit status of the last command run.
E. Section: Comments, subsection: Implementation Comments: Comment tricky, non-obvious, interesting or important parts of your code.
F. Section: Comments, subsection: TODO Comments: Use TODO comments for code that is temporary, a short-term solution, or good-enough but not perfect.
F.0 TODOs should include the string TODO in all caps, followed by the handle of the person with the best context about the problem referenced by the TODO.
G. Section: Formatting, subsection: Indentation: Indent 2 spaces. No tabs. (notepad++ ctrl+h (open replace), alt+x (enable extended search/replace) replace "\t" with:"  " (zonder de quotations)).
I. Section: Formatting, subsection: Line Length: Maximum line length is 80 characters.
J. Section: Pipelines: Pipelines should be split one per line if they don’t all fit on one line.
Etc. 

You can then select a file, then list the methods and complete the checklist per function. (I think, if you do all todo's per function, you will get a better/deeper insight in the functions than when you apply the same improvement to each function, then do the next improvement. However, if you like the latter better, feel free).

On top of that, shellcheck compliance can be realised: https://github.com/koalaman/shellcheck 
0. Open terminal.
1. cd (change directory) to the path where you have a bash file that you want to make shellcheck compliant. e.g. 
```
cd c:/some_folder/
```
2. Verify it contains the file you want: e.g. `dir` should return:
```
the_bash_file_you_want_to_make_shellcheck_compliant.sh
```
3. Type:
```
shellcheck the_bash_file_you_want_to_make_shellcheck_compliant.sh
```
4. enter.
5. Then you get a list of things that are not shellcheck compliant.
6. Then you look up what the shellcheck issue number means. 
7. Then you do the magic=resolve it intelligently to make it shellcheck compliant. For example, put variable inside double quotes.
7.1 If making it shellcheck compliant would require an undesired change of the code, you could include a message for shellcheck to ignore that specific incompliance https://github.com/koalaman/shellcheck/wiki/Ignore:
```
# shellcheck disable=SC2116
hash=$(echo ${hash})    # trim spaces
```
8. Then you run shellcheck again on the same file to verify you have made it fully shellcheck compliant =no more warnings/errors.

Ideally, you assign yourself to the issue (in this case, give me a headsup on which file you want to do, so that we do not simultaneously work in the same file), create a branch, apply the fixes, and send me a pull request.

### PR DESCRIPTION
**Pull request by mohitsaxenaknoldus**
Fixes #27

### CODE REVIEW

File: src/push_gitlab_build_status_to_github_website.sh [hunk 1]
```diff
@@ -47,28 +51,28 @@ get_and_export_build_status_to_github_build_status_website_repo() {
 	pipelines=$(curl --header "PRIVATE-TOKEN: $personal_access_token" "http://127.0.0.1/api/v4/projects/$gitlab_username%2F$repository_name/pipelines")
 	
 	# get build status from pipelines
-	job=$(echo $pipelines | jq -r 'map(select(.sha == "'"$branch_commit"'"))')
-	status=$(echo "$(echo $job | jq ".[].status")" | tr -d '"')
+	job=$(echo "$pipelines" | jq -r 'map(select(.sha == "'"$branch_commit"'"))')
+	status=$(cmd "$(echo "$job" | jq ".[].status")" | tr -d '"')
```
Comment (a-t-0) [inline src/push_gitlab_build_status_to_github_website.sh:55]: Could you explain what the difference is between using `echo` and `cmd`?  I'm simply curious :)

**Review by a-t-0 — APPROVED** [review]

File: src/push_gitlab_build_status_to_github_website.sh (hunk 1 above)
Comment (mohitsaxenaknoldus) [reply src/push_gitlab_build_status_to_github_website.sh:55]: Previously, it was supplying an `echo` to an `echo` where the second `echo` was a normal *command* so it's better to just use `cmd`.

**Comment by a-t-0** [comment]
Marvellous work! You did that quite fast. Your initial comment on the PR `fixes #27` indicates I was not clear on the issue. On top of that, the initial title was `shellcheck compliance` instead of `Style Guide Compliance`.  I updated the issue to reflect that `Shellcheck Compliance` is a subset of Google Style Guide Compliance.

Your PR is accepted! 

Would you like me to make separate issues for Google Style Guide compliance per file?

File: src/push_gitlab_build_status_to_github_website.sh (hunk 1 above)
Comment (a-t-0) [reply src/push_gitlab_build_status_to_github_website.sh:55]: Thank you for your explanation!

**Pull request merged**